test.py

```diff
@@ -51,13 +51,15 @@ def on_receive_message(topic, message):
 tls_options = None
 if args.cert or args.key or args.root_ca:
     if args.cert:
-        assert(args.key)
-        tls_options = io.TlsContextOptions.create_client_with_mtls(args.cert, args.key)
+        assert (args.key)
```

Comment:
remove parens, I believe. otherwise this evaluates to a tuple with one element in it, which is truthy regardless of the element's value

Reply:
I wondered about that, I didn't put them there, but I am definitely removing them.